projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -202,6 +202,12 @@ public function add_tools_menu( $wp_admin_import = false, $wp_admin_export = fal
 	public function add_options_menu( $wp_admin = false ) {
 		add_options_page( esc_attr__( 'Hosting Configuration', 'jetpack' ), __( 'Hosting Configuration', 'jetpack' ), 'manage_options', 'https://wordpress.com/hosting-config/' . $this->domain, null, 6 );
 
+		// No need to add a menu linking to WP Admin if there is already one.
+		if ( ! $wp_admin ) {
```

Comment:
Let's also wait for @obenland to weigh in https://github.com/Automattic/jetpack/pull/18805#discussion_r576065718, since we may want to always link to both Calypso and WP Admin regardless the link destination toggle.

Reply:
Thanks @mmtr. I think in your case with the user management and SSO it makes sense because of the feature discrepancy. But in this case is there any reason why we would show "Advanced General" and "Advanced Writing" alongside "General" and "Writing" if WP Admin links are enabled and they are essentially duplicates sending the user to the same URL?

Reply:
> is there any reason why we would show "Advanced General" and "Advanced Writing" alongside "General" and "Writing"

If a user wants to delete their site, they can only do that within Calypso. I think that feature is important enough for always adding a "General" link to Calypso.

Reply:
@mmtr yes great point, I had not considered that. Thanks for pointing it out!

Reply:
Anyway, the overall agreement as noted in paYJgx-1lp-p2#comment-1661 is that we don't want to include escape hatches in order to keep all features accessible, and we should always rely on the link destination toggle. 

So, on that note, I think this PR is good as-is. We can always come back and adjust if necessary.